client/app/account/page.tsx

```diff
@@ -1,19 +1,24 @@
 'use client'
 
 import React from 'react'
+import { useRouter } from 'next/navigation'
 import { FiEdit } from 'react-icons/fi'
 import { FormContainer } from '@/_components/layout'
 import { Button, Heading } from '@/_components/ui'
 import { AccountInfo } from './component'
 
 export default function Account() {
+  const router = useRouter()
```

Comment:
Can you add one empty line below this for better reading? 🙏

Reply:
Fixed it. 
b23466c8ecaed82d91aa9509eecd76ee14937d34